src/lib/readdirRecursive.ts

```diff
@@ -21,7 +21,7 @@ export default function readdirRecursive(folderToSearch: string, ignoreGit = fal
   let ignoreFilter: Ignore;
   if (ignoreGit) {
     // Initialize ignore filter with `.git` directory
-    ignoreFilter = ignore().add(path.join(folderToSearch, '.git/'));
+    ignoreFilter = ignore().add(path.join(folderToSearch, '.git', path.sep));
```

Comment:
There seems to be a bug in `ignore` where `.git/` doesn't get escaped on Windows.

https://github.com/kaelzhang/node-ignore/issues/81

![200710606-07b1cd56-6a97-446e-ac40-7f955e8fadb3](https://user-images.githubusercontent.com/33762/201496573-f1778318-832c-445e-8d19-49c82090bea0.png)